README.md

```diff
@@ -53,6 +53,8 @@ inactive: 5
 expire: 6
 ```
 
+To define `last_changed` with a value equal to current days from 1970 use `-1`.
```

Comment:
Can you make it `now` instead? it's more explicit on what it does

Reply:
done